SimplifiedTests/NYPLBookmarkSpec+AudiobookTests.swift

```diff
@@ -1,137 +0,0 @@
-//
```

Comment:
@ettore This file is being removed because it was migrated to NYPLAudiobookToolkit, I just forgot to remove it in previous PR

Reply:
thank you for being thorough @ErnestFan  -- it's really difficult to keep track for things that are missing or should be missing.